tests/api_tests/test_questionnaire_response_api.py

```diff
@@ -526,8 +526,7 @@ def test_gror_consent(self):
                            resource,
                            expected_status=http.client.BAD_REQUEST)
 
-    @mock.patch('rdr_service.dao.bq_participant_summary_dao.logging')
-    def test_gror_consent_with_duplicate_answers(self, mock_logging):
+    def test_gror_consent_with_duplicate_answers(self):
```

Comment:
I think we can replace this with a mock of the resource generator logging.  The intent was to confirm that the generator logged the message about skipping duplicate responses to the same question.

Reply:
Cool, I wasn't quite how to update that mock patch.